config.toml

```diff
@@ -57,50 +57,78 @@ pluralizeListTitles = false
   name = "About"
   url = "/about/"
   weight = 2
-  identifier = "about"
 
 [[menu.main]]
   name = "Resources"
-  url = "/resources"
+  url = "/resources/"
   weight = 3
 
+[[menu.main]]
+  name = "FAQ"
+  url = "/about/faq/"
+  weight = 4
+
+#######################
```

Comment:
We shouldn't include a comment like this that isn't present in other sites. If we need to split the configs, we can look at doing that as it is [supported by Hugo](https://gohugo.io/getting-started/configuration/#configuration-directory), and we've done it in the Hugo Solstice Theme exampleSite configuration.